core/templates/dev/head/galleries/Gallery.js

```diff
@@ -436,11 +481,11 @@ oppia.controller('SearchBar', [
     );
 
     $scope.showCreateExplorationModal = function() {
-      ExplorationCreationButtonService.showCreateExplorationModal(
+      createExplorationButtonService.showCreateExplorationModal(
```

Comment:
there was an issue of creating new explorations, where ExplorationCreationButtonService was undefined on my machine (rendering new exploration creation useless), changing it to createExplorationButtonService resolved the issue.